### PR TITLE
Adding tinymce for all text areas.

### PR DESCRIPTION
**Pull request by awais786**
Implement tinymce for multiple fields.

https://openedx.atlassian.net/browse/ECOM-7567

### CODE REVIEW

**Comment by awais786** [comment]
**Input** 

screen shot 2017-04-04 at 12 38 22 pm

**Rendering**

screen shot 2017-04-04 at 12 38 11 pm

**Review by tasawernawaz — APPROVED** [review]

**Comment by awais786** [comment]
In case of Max-length issue. Page will show the error.
screen shot 2017-04-07 at 3 06 04 pm

File: bower.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "moment": "~2.13.0",
     "pikaday": "https://github.com/owenmead/Pikaday.git#1.4.0",
     "underscore": "~1.8.3",
-    "google-diff-match-patch": "0.1.0"
+    "google-diff-match-patch": "0.1.0",
+    "tinymce": "4.5.5"
```
Comment (waheedahmed) [inline bower.json]: Update to latest version `4.5.6`.

**Review by waheedahmed — APPROVED** [review]
Just one comment.

**Pull request merged**